scripts/configure_klp

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import click
+
+from qp_klp import plugin
+
+
+@click.command()
+@click.option('--env-script', prompt='Environment script',
+              default='source activate qp-klp')
+@click.option('--server-cert', prompt='Server certificate', default='None')
+def config(env_script, server_cert):
+    """Generates the Qiita configuration files"""
+    if server_cert == 'None':
+        server_cert = None
```

Comment:
```suggestion
```

...if `server_cert` has `required=False`, it defaults to `None`

Reply:
I had to check this because it certainly is a confusing situation, but `required=False` won't have an effect on this prompt unless it is paired with a `default` value. Unfortunately if you set the default value to `None` then no default value is set. Hence why the default value here would have to be `'None'`. I've added a default value to the option declaration above which will make this clearer.